src/stories/components/HorizontalTitle/HorizontalTitle.js

```diff
@@ -0,0 +1,45 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+/**
+ * HorizontalTitle Component
+ *
+ * @component
+ * @param {eyebrow} eyebrow
+ * @param {title} title
+ * @param {content} paragraph content
+ * @return {object}
+ */
+export const HorizontalTitle = (
+    { modifierClasses, eyebrow, title, content }) => {
+    return (
+        <div
+            className={
+                ['horizontal-title', `${modifierClasses}`].join(' ').trim()
+            }
+        >
+            {eyebrow ?
+                <div className="horizontal-title__eyebrow">{eyebrow}</div> : ''
+            }
+            <h1 className="horizontal-title__title">{title}</h1>
+            <p className="horizontal-title__content">{content}</p>
```

Comment:
Paragraphs shouldn't be nested inside of each other. Either convert this to a `div` or pass `{content}` as a string literal.